.changeset/lovely-peaches-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights': minor
```

Comment:
We are still pre-1.0 on this package so you can keep this bump as patch instead.